tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "downlevelIteration": true,
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
+    "skipLibCheck": true,
```

Comment:
where did this come from

Reply:
I added it as a fix

Reply:
After some node changes on the CI.
I also rebuilded the package-lock with the correct version of node because it was causing issues as it was last build with 14.16.0 and on the CI it is 10.15.3

Reply:
It can be removed after a successful upgrade of Node on the CI

Reply:
We should not do this. Clients would not be able to npm i the repo if they have newer node version

Reply:
should be fixed in patches as well

Reply:
It is fixed in patches

Reply:
nope, it will be obsolete after upgrade to node 14 or above